pkgs/top-level/all-packages.nix

```diff
@@ -13185,9 +13183,7 @@ with pkgs;
 
   fira-code = callPackage ../data/fonts/fira-code { };
   fira-code-symbols = callPackage ../data/fonts/fira-code/symbols.nix { };
-  fira-code-nerdfont = nerdfonts.override {
-    fonts = [ "FiraCode" ];
-  };
+  fira-code-nerdfont = nerd-fonts.fira-code;
```

Comment:
Hmm maybe put this and the other alike changes in `aliases.nix`? Perhaps even with a `lib.warn`?